core/blockchain.go

```diff
@@ -1123,7 +1122,7 @@ func (bc *BlockChain) insertChain(chain types.Blocks) (int, []interface{}, []*ty
 	for i, block := range chain {
 		// If the chain is terminating, stop processing blocks
 		if atomic.LoadInt32(&bc.procInterrupt) == 1 {
-			log.Debug("Premature abort during blocks processing")
+			utils.GetLogger().Debug("Premature abort during blocks processing")
```

Comment:
These change on ethereum's log code will potentially add a lot more logs for us. Can you test the logs before and after this change and see how much difference in the size?

Reply:
Ran both versions for 10 minutes, barely any increase:
• Before: ~2.0M
• After: ~2.1M
Seems like most of these are not called or are `Error` log types (which shouldn't happen often).